shader/shared/config.wgsl

```diff
@@ -49,3 +49,12 @@ let N_TILE_Y = 16u;
 let N_TILE = 256u;
 
 let BLEND_STACK_SPLIT = 4u;
+
+// Radial gradient kinds
```

Comment:
I would add a comment: "These are computed in draw_leaf from the generic gradient parameters encoded in the scene, and stored in the gradient's info struct, for consumption during fine rasterization."